gkeep2notion.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/local/bin/python3
+from typing import List, Dict
```

Comment:
No need to do this for typing since Python 3.9: https://peps.python.org/pep-0585/#implementation

Reply:
I believe I need that one, because of the Docker container I'm running the script in

Reply:
I see, that Dockerfile uses Python 3.8. No worries then.